ros_ign_bridge/src/service_factories/ros_ign_interfaces.cpp

```diff
@@ -44,25 +44,41 @@ get_service_factory__ros_ign_interfaces(
         ignition::msgs::Boolean>
     >(ros_type_name, "ignition.msgs.WorldControl", "ignition.msgs.Boolean");
   }
+
+  // NOTE(CH3): Prioritising ignition since versions prior to Garden cannot support gz
```

Comment:
How about we keep using `ignition` for now, which will result in warnings in Garden but hopefully goes through? And when we backport whatever we can to Citadel and Fortress, we flip the switch here and use `gz` for all versions.

Reply:
Okay, note that the `ignition` space will only work if the `include/ignition` version of the message header is used, which... should be fine I think?

Reply:
Ah, I didn't notice the message generation workflow changed. Hopefully it works: https://github.com/gazebosim/ros_gz/pull/280/commits/1451eb70bc7bd8d8f9cb02eef54ba93fdf2d57f7